.github/workflows/_shared-docs-build-pr.yml

```diff
@@ -108,6 +108,13 @@ on:
         description: A newline separated list of link targets that should not cause reference errors. A small RST file will be created during the build which contains these labels.
         required: false
         type: string
+      intersphinx-links:
+        description: |
+          A newline separated list of entries to add to the intersphinx_mapping in the generated conf.py.
+          Use the syntax 'identifier:https://server/path' to add the identifier 'identifier' with URL
+          'https://server/path'.
```

Comment:
As far as I know there is no intrinsic markdown support for these descriptions.
It may render in our wiki entries, but that is more of a side effect of my crude "action/workflow to MD" generation than a feature.